### PR TITLE
Add Kubernetes multi-cluster support

### PR DESCRIPTION
**Pull request by herbergt**
* Added Kubernetes multi-cluster support.
* Make fraction digits configurable
* Add additional query parameters
* Make the base URL configurable to be shown in the Kubecost plugin

### CODE REVIEW

**Comment by herbergt** [comment]
@phac008 I would like to contribute with these changes to this project.

File: src/constants.ts [hunk 1]
```diff
@@ -18,4 +18,4 @@
  * @public
  */
 
-export const KUBECOST_ANNOTATION_DEPLOYMENT_NAME = 'kubecost.com/deployment-name';
\ No newline at end of file
+export const KUBECOST_DEFAULT_ANNOTATION_DEPLOYMENT_NAME = 'kubecost.com/deployment-name';
```
Comment (phac008) [inline src/constants.ts:21]: Variable needs to be adapted in this files too:

src/components/useAppData.ts
src/conditions.ts

**Review by phac008 — CHANGES_REQUESTED** [review]
hi!

great! thank you for contribution :-)
please see my comments in src/constants.ts

regards
Philipp

File: src/constants.ts (hunk 1 above)
Comment (herbergt) [reply src/constants.ts:21]: Thanks for pointing that out @phac008, the updates has been done.

**Pull request merged**